debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -122,9 +190,7 @@
 #djDebug button:active {
     border: 1px solid #aaa;
     border-bottom: 1px solid #888;
-    box-shadow:
-        inset 0 0 5px 2px #aaa,
-        0 1px 0 0 #eee;
+    box-shadow: inset 0 0 5px 2px #aaa, 0 1px 0 0 #eee;
```

Comment:
This doesn't look as if it has been formatted by prettier v4 (using `pre-commit`)?

Reply:
probably was my VS formatter, odd since it's set to use prettier (maybe was not using the same rules)
![image](https://github.com/jazzband/django-debug-toolbar/assets/75857767/e13c2dd3-6677-473c-8d4a-ec16a168504f)
My last commit reverted that change